src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java

```diff
@@ -63,6 +64,14 @@ public void validateUserIsPresent(Optional<UserProfile> userProfile) {
         }
     }
 
+    public void validateUserAlreadyExists(Optional<UserProfile> userProfile) {
+        if (!userProfile.isEmpty()) {
+            final String exceptionMsg = "RequiredFieldMissingException - The "
+                    + "userIdam Id you are tryign to update already exists";
+            exceptionService.throwCustomRuntimeException(RESOURCEALREADYEXISTS, exceptionMsg);
```

Comment:
switch case is missing for custom exception in ExceptionServiceImpl

Reply:
Added case RESOURCENOTFOUNDEXCEPTION